src/components/GoodList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const GoodList = ({ goods }) => (
+  <ul>
+    {goods.map(good => (<li key={good}>{good}</li>))}
```

Comment:
```suggestion
    {goods.map(good => <li key={good}>{good}</li>)}
```
If it returns some value in one line you can get rid of parentheses